CommandHelper/LocalPackages/commands/chars/cloak.yml

```diff
@@ -0,0 +1,41 @@
+class: cloak
```

Comment:
You put the cloak stuff in with the kilmonger stuff, i think you need to redo killmonger, cuz im not sure how u would remove cloak from this PR

thhanks